samples/excel/89-preview-apis/protect-data-in-worksheet-and-workbook-structure.yaml

```diff
@@ -18,100 +18,145 @@ script:
 
         async function protectDataInWorksheet() {
             await Excel.run(async (context) => {
-                await worksheetHelper(context, true, false);
+                let activeSheet = context.workbook.worksheets.getActiveWorksheet();
+                activeSheet.load("protection/protected");
+
+                await context.sync();
+
+                if (!activeSheet.protection.protected) {
+                    activeSheet.protection.protect();
+                }
+
+                activeSheet.load("name");
+                activeSheet.load("protection/protected");
+
+                await context.sync();
+
+                console.log("Expected: Worksheet '" + activeSheet.name + "' should " + messageHelper(true) + ". Protected? " + activeSheet.protection.protected);
             });
         }
 
         async function unprotectDataInWorksheet() {
             await Excel.run(async (context) => {
-                await worksheetHelper(context, false, false);
+                let activeSheet = context.workbook.worksheets.getActiveWorksheet();
+                activeSheet.protection.unprotect();
+
+                activeSheet.load("name");
+                activeSheet.load("protection/protected");
+
+                await context.sync();
+
+                console.log("Expected: Worksheet '" + activeSheet.name + "' should " + messageHelper(false) + ". Protected? " + activeSheet.protection.protected);
             });
         }
 
         async function protectWorkbookStructure() {
```

Comment:
For the workbook structure functions, instead of having console.log that tells people what happened, give instructions in the page just above the button that tells them how to verify. E.g., have a brief paragraph that says "Click the next button and then notice that you cannot delete or add a new worksheet."